tests/test_model.py

```diff
@@ -288,7 +288,7 @@ def command_2(opts):
                     'default': '',
                     'minLength': 0,
                     'type': 'text',
-                    'name': 'text',
+                    'name': 'text_array_item',
                     'description': None
```

Comment:
Because we used to have default `name`s for data types the name value of a `text` item in an `array` would be `"text"`. The name is now `text_array_item`, `number_array_item`, etc... but maybe it should just stay `None`.